Update docs Tools -> Plugins (#130)
hedera-dev/hedera-agent-kit: typescript/README.md

npm install hedera-agent-kit @langchain/openai @langchain/core langchain @hashgraph/sdk dotenv
```

> > You may need to add in the "type":"module" to your package.json file after `"scripts"` in order tospecify that the project is using ES modules

Reviewer: ```suggestion
> You may need to add "type": "module" to your package.json file so that your project allows ES module imports
```
